src/surfaces/filtersRail/filtersRail.jsx

```diff
@@ -285,7 +290,7 @@ function FiltersRail(props) {
                             <div className={actionButtonsClasses}>
                                 <A
                                     className={clearFiltersClasses}
-                                    disable={!filterOptions.isDirty}
+                                    disable={!filterOptions.isDirty || !filterOptions.isFiltering}
```

Comment:
Is OR correct here?  Or should it be AND?
Also, can we use `disabled` rather than `disable`, or does `<A>` component only use `disable`?

Reply:
`OR` is the correct condition applied. `<A>` component only use `disable` 😞

Reply:
Got it.  Thanks for confirming.  I think it's so many NOTs, coupled with the fact that it's the condition for with the Link Button should NOT be active (i.e. should be disabled) -- that what made it slightly tricky for me. =)  But I think it makes sense.